jsettlers.graphics/src/main/java/jsettlers/graphics/map/draw/settlerimages/SettlerImageMap.java

```diff
@@ -38,40 +36,31 @@
  */
 public final class SettlerImageMap {
 
-	private static final SettlerImageMapItem DEFAULT_ITEM = new SettlerImageMapItem(10, 0, 0, 1);
+	static final SettlerImageMapItem DEFAULT_ITEM = new SettlerImageMapItem(10, 0, 0, 1);
 
 	private static SettlerImageMap instance;
 
-	private final ImageProvider imageProvider = ImageProvider.getInstance();
+	private final ImageProvider imageProvider;
 
-	private final SettlerImageMapItem[][][][] map;
+	final HashMap<SettlerImageFlavor, SettlerImageMapItem> map = new HashMap<>();
```

Comment:
private

Reply:
It's used to assert contents in the map in the class test.